src/openzaak/management/commands/generate_data.py

```diff
@@ -64,6 +65,19 @@
     ZaakInformatieObjectFactory,
     ZaakObjectFactory,
 )
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+
+@vcr.use_cassette("src/openzaak/tests/cassettes/selectielijst_resultaten.yaml")
```

Comment:
Any reason why we are using vcr to create a cassette? I don't see us using this for anything atm.

Reply:
I've read vcr documentation one more time and noticed that they recommend to use it with tests. There is no example o use vcr outside of tests. So after some consideration I decided not to use it. So let's just call Selectielijst API resource for real in this command